### PR TITLE
Proofreading c5

### PR DESCRIPTION
**Pull request by jannes-m**

### CODE REVIEW

File: 05-geometry-operations.Rmd [hunk 1]
```diff
@@ -20,7 +20,7 @@ library(spDataLarge)
 
 So far the book has explained the structure of geographic datasets (Chapter \@ref(spatial-class)), and how to manipulate them based on their non-geographic attributes (Chapter \@ref(attr)) and spatial relations (Chapter \@ref(spatial-operations)).
 This chapter focuses on manipulating the geographic elements of spatial objects, for example by creating buffers, simplifying and converting vector geometries, and aggregating and resampling raster data.
-After reading it --- and attempting the exercises at the end --- you should understand and have control over the geometry column in `sf` objects and the extent and geographic location of pixels represented in rasters in relation to other geographic objects.
+After reading it --- and attempting the Exercises at the end --- you should understand and have control over the geometry column in `sf` objects and the extent and geographic location of pixels represented in rasters in relation to other geographic objects.
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd:23]: Really? No objections, the capitalisation highlights them, but not sure it's necessary. No biggy either way, we should check to see if this applied consistently.

File: 05-geometry-operations.Rmd [hunk 2]
```diff
@@ -51,7 +51,7 @@ Another reason for simplifying objects is to reduce the amount of memory, disk s
 it may be wise to simplify complex geometries before publishing them as interactive maps. 
 The **sf** package provides `st_simplify()`, which uses the Douglas-Peucker algorithm to reduce the vertex count.
 `st_simplify()` uses the `dTolerance` to control the level of generalization in map units [see @douglas_algorithms_1973 for details].
-Figure \@ref(fig:seine-simp) illustrates simplification of a `LINESTRING` geometry representing the river Seine and tributaries.
+Figure \@ref(fig:seine-simp) illustrates simplification of a `LINESTRING` geometry representing the River Seine and tributaries.
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd:54]: :+1:

File: 05-geometry-operations.Rmd [hunk 3]
```diff
@@ -89,8 +89,8 @@ This means the 'topology' is lost, resulting in overlapping and 'holey' areal un
 By default it uses the Visvalingam algorithm, which overcomes some limitations of the Douglas-Peucker algorithm [@visvalingam_line_1993].
 <!-- https://bost.ocks.org/mike/simplify/ -->
 The following code chunk uses this function to simplify `us_states`.
-The result has only 1% of the vertices of the input (set using the argument `keep`) but its number of objects remains intact because we set `keep_shapes = TRUE`:^[
-Simplification of multipolygon objects can remove small internal polygons, even if the `keep_shapes` argument is set to TRUE. To prevent this, you need to set `explode = TRUE`. This option converts all mutlipolygons into separate polygons before its simplification.
+The result has only 1% of the vertices of the input (set using the argument `keep`), but its number of objects remains intact because we set `keep_shapes = TRUE`:^[
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd:92]: :+1:

File: 05-geometry-operations.Rmd [hunk 4]
```diff
@@ -300,15 +300,15 @@ rotation = function(a){
 } 
 ```
 
-The `rotation` function accepts one argument `a` - a rotation angle in degrees.
+The `rotation` function accepts one argument `a` --- a rotation angle in degrees.
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd:303]: :+1:

File: 05-geometry-operations.Rmd [hunk 5]
```diff
@@ -375,8 +375,8 @@ op = par(mar = rep(0, 4))
 x = b[1]
 y = b[2]
 x_and_y = st_intersection(x, y)
-plot(b, border = "grey")
-plot(x_and_y, col = "lightgrey", border = "grey", add = TRUE) # intersecting area
+plot(b, border = "gray")
+plot(x_and_y, col = "lightgray", border = "gray", add = TRUE) # intersecting area
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd:379]: I thought there was an issue but testing suggests it's OK:

``` r
plot(1:9, col = "gray")
```



<sup>Created on 2024-09-28 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Comment (Robinlovelace) [reply 05-geometry-operations.Rmd:379]: Do we now use "gray" throughout?

File: 05-geometry-operations.Rmd [hunk 6]
```diff
@@ -799,18 +799,18 @@ The extent was slightly adjusted to accommodate the new grid size.
 ```{r aggdf}
 #| echo: false
 agg_df = data.frame(
-  object = c("dem", "dem_agg"),
-  resolution = c("(30.85, 30.85)", "(154.25, 154.25)"),
-  dimensions = c("117 x 117", "24 x 24"),
-  extent = c("794599.1, 798208.6, 8931775, 8935384", "794599.1, 798301.1, 8931682, 8935384")
+  Object = c("dem", "dem_agg"),
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd:802]: :+1:

File: _05-ex.Rmd [hunk 7]
```diff
@@ -118,8 +118,8 @@ arrange(us_states_bor, -borders)
 ```
 
 E7. Read the srtm.tif file into R (`srtm = rast(system.file("raster/srtm.tif", package = "spDataLarge"))`).
-This raster has a resolution of 0.00083 by 0.00083 degrees. 
-Change its resolution to 0.01 by 0.01 degrees using all of the method available in the **terra** package.
+This raster has a resolution of 0.00083 * 0.00083 degrees.
```
Comment (Robinlovelace) [inline _05-ex.Rmd:121]: Really? cc @Nowosad

**Review by Robinlovelace — APPROVED** [review]
Some definite fixes in there. Will merge. 

Some questions to revisit though cc @Nowosad see comments and let's do a last sweep with global search/replace for consistency after all chapters reviewed individually.

File: _05-ex.Rmd (hunk 7 above)
Comment (Nowosad) [reply _05-ex.Rmd:121]: I prefer the previous version

**Pull request merged**